<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>KLoanDetailsPage</class>
  <widget class="QWidget" name="KLoanDetailsPage">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>592</width>
        <height>434</height>
      </rect>
    </property>
    <layout class="QVBoxLayout">
      <item>
        <widget class="QLabel" name="qLabel2">
          <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
            <string>Now enter the details for your loan. You can leave one of the fields empty and KMyMoney will calculate it when you press the &lt;b&gt;Calculate&lt;/b&gt; button. Before you can continue with the next page you also need to press &lt;b&gt;Calculate&lt;/b&gt; so that KMyMoney can check the logical correctness of the values you have supplied.</string>
          </property>
          <property name="wordWrap">
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <item>
        <spacer name="spacer89">
          <property name="sizeHint">
            <size>
              <width>20</width>
              <height>70</height>
            </size>
          </property>
          <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <item>
            <layout class="QGridLayout">
              <item row="0" column="0">
                <widget class="QLabel" name="textLabel7">
                  <property name="text">
                    <string>The interest rate gets calculated</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item row="0" column="1" rowspan="1" colspan="3">
                <widget class="KComboBox" name="m_paymentDue">
                  <item>
                    <property name="text">
                      <string>when the payment is received</string>
                    </property>
                  </item>
                  <item>
                    <property name="text">
                      <string>when the payment is due</string>
                    </property>
                  </item>
                </widget>
              </item>
              <item row="1" column="0">
                <widget class="QLabel" name="textLabel8">
                  <property name="text">
                    <string>Loan amount</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item row="1" column="1" rowspan="1" colspan="3">
                <widget class="AmountEdit" name="m_loanAmount">
                  <property name="resetButtonVisibility" stdset="0">
                    <bool>true</bool>
                  </property>
                  <property name="whatsThis" stdset="0">
                    <string>Enter the amount of the loan in this field. If you want to calculate this field out of the other parameters, please leave it empty. If the field is marked as required (colored background) you have informed KMyMoney about the fact that there were already some payments towards the loan. In this case, please enter the ending balance of your last statement.</string>
                  </property>
                </widget>
              </item>
              <item row="2" column="0">
                <widget class="QLabel" name="textLabel9">
                  <property name="text">
                    <string>Interest rate</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item row="2" column="1" rowspan="1" colspan="3">
                <widget class="AmountEdit" name="m_interestRate">
                  <property name="whatsThis" stdset="0">
                    <string>Please enter the interest rate (as a percentage) or leave the field empty to calculate it.</string>
                  </property>
                </widget>
              </item>
              <item row="3" column="0">
                <widget class="QLabel" name="textLabel10">
                  <property name="text">
                    <string>Term</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item row="3" column="1" rowspan="1" colspan="3">
                <layout class="QHBoxLayout">
                  <item>
                    <widget class="QSpinBox" name="m_termAmount">
                      <property name="maximum">
                        <number>999</number>
                      </property>
                      <property name="whatsThis" stdset="0">
                        <string>Please enter the term of this loan or enter 0 to calculate it. The term is the time that is required to fully repay the loan. This time might be different from the time your loan contract is signed for.</string>
                      </property>
                    </widget>
                  </item>
                  <item>
                    <widget class="KMyMoneyFrequencyCombo" name="m_termUnit"/>
                  </item>
                </layout>
              </item>
              <item row="4" column="0">
                <widget class="QLabel" name="textLabel11">
                  <property name="text">
                    <string>Payment (principal and interest)</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item row="4" column="1" rowspan="1" colspan="3">
                <widget class="AmountEdit" name="m_paymentAmount">
                  <property name="whatsThis" stdset="0">
                    <string>Please enter the amount you pay for principal and interest or leave the field empty to calculate it.</string>
                  </property>
                </widget>
              </item>
              <item row="5" column="0">
                <widget class="QLabel" name="textLabel12">
                  <property name="text">
                    <string>Balloon payment</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item row="5" column="1" rowspan="1" colspan="3">
                <widget class="AmountEdit" name="m_balloonAmount">
                  <property name="whatsThis" stdset="0">
                    <string>Please enter the amount of a final amortization payment or leave the field empty to calculate it.</string>
                  </property>
                </widget>
              </item>
              <item row="6" column="1">
                <layout class="QHBoxLayout">
                  <item>
                    <spacer name="spacer86">
                      <property name="sizeHint">
                        <size>
                          <width>71</width>
                          <height>20</height>
                        </size>
                      </property>
                      <property name="sizeType">
                        <enum>QSizePolicy::Expanding</enum>
                      </property>
                      <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                      </property>
                    </spacer>
                  </item>
                  <item>
                    <widget class="QPushButton" name="m_calculateButton">
                      <property name="text">
                        <string>Calculate</string>
                      </property>
                      <property name="toolTip" stdset="0">
                        <string>Press this button to calculate/verify your loan details.</string>
                      </property>
                      <property name="whatsThis" stdset="0">
                        <string>Pressing this button calculates a possibly missing parameter for your loan or verifies that the values entered match. If something is not correct you will receive information about it.</string>
                      </property>
                    </widget>
                  </item>
                </layout>
              </item>
              <item row="6" column="2">
                <spacer name="spacer88_2">
                  <property name="sizeHint">
                    <size>
                      <width>20</width>
                      <height>16</height>
                    </size>
                  </property>
                  <property name="sizeType">
                    <enum>QSizePolicy::Expanding</enum>
                  </property>
                  <property name="orientation">
                    <enum>Qt::Vertical</enum>
                  </property>
                </spacer>
              </item>
            </layout>
          </item>
          <item>
            <spacer name="spacer87">
              <property name="sizeHint">
                <size>
                  <width>27</width>
                  <height>21</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
      <item>
        <spacer name="spacer88">
          <property name="sizeHint">
            <size>
              <width>20</width>
              <height>16</height>
            </size>
          </property>
          <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
    <customwidgets>
    <customwidget>
      <class>KComboBox</class>
      <extends>QComboBox</extends>
      <header location="global">kcombobox.h</header>
    </customwidget>
  <customwidget>
   <class>QSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>                                                                                                                           
   <class>KMyMoneyFrequencyCombo</class>                                                                                                   
   <extends>QWidget</extends>                                                                                                              
   <header>kmymoneyfrequencycombo.h</header>
   <container>1</container>                                                                                                                
  </customwidget>   
    <customwidget>
      <class>AmountEdit</class>
      <extends>QLineEdit</extends>
      <header location="global">amountedit.h</header>
    </customwidget>
 </customwidgets>
</ui>
